docs/Early Access - Using Global Profile Configuration.md

```diff
@@ -53,6 +53,19 @@ The global profile functionality simplifies profile management by enabling you t
 
 **Note**: A project context takes precedence over global configuration.
 
+### Create a Team Configuration File
+
+Create a team configuration file.
+
+1. Navigate to the explorer tree.
+2. Hover over **DATA SETS**, **USS**, or **JOBS**.
+3. Click the **+** icon.
+4. Select **Create a New Team Configuration File**.
+5. Follow the instructions to complete the team configuration profile creation.
```

Comment:
I am not sure about the # 5 here, are there instructions to follow or would it just be choose global or workspace config file?